src/contributors/RebaiAhmed.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "Rebai Ahmed",
+    "desc": "I am Full Stack Developer ,Open source Enthusiast , TunisianJs-Community Co-Founder , Meetups Co-Organizer",
+    "github": "https://github.com/Rebaiahmed",
+    "twitter": "https://twitter.com/RebaiAhmed_",
+    "linkedin": "https://www.linkedin.com/in/rebaiahmed",
+    "stackoverflow": "https://stackoverflow.com/users/5330655/rebai-ahmed",
+    "quora": "https://www.quora.com/profile/Rebai-Ahmed",
+    "medium" :"https://ahmedrebai.medium.com",
+    "portfolio":"https://rebaiahmed.github.io"
```

```suggestion
    "linkedin": "https://www.linkedin.com/in/rebaiahmed"
```
Thanks for adding all of your socials, appreciated. Unfortunately, we have options to just add `github`, `twitter` & `linkedin`. Please update it

---

I just updated it :)